neuralforecast/models/gru.py

```diff
@@ -199,6 +218,7 @@ def forward(self, windows_batch):
         hidden_state, _ = self.hist_encoder(
             encoder_input
         )  # [B, seq_len, rnn_hidden_state]
+        hidden_state = self.encoder_activation(hidden_state)
```

Comment:
The encoder activation should not be set in .forward here, but in the definition of self.hist_encoder - see the arguments supported by GRU

Reply:
From the [documentation](https://pytorch.org/docs/stable/generated/torch.nn.GRU.html), there is no parameter for specifying an activation function, which is why I add it in the forward method.

Reply:
Actually, maybe we should just close this PR and the related issue, and say that it's not possible to change the activation functions inside of GRU. 

I'd say we have two options:

1. Remove the parameter and use the default activation from PyTorch, which they froze and we can't modify.
2. Implement a custom GRU class where we can change the activation function (not sure if it's worth it, especially since it's not that performant for forecasting).

What do you think?